src/main/java/techreborn/compatmod/ic2/IC2Dict.java

```diff
@@ -149,11 +184,27 @@ public static void init() {
 		}
 	}
 
-	public static ItemStack getIC2Cable(CableType type, int insulation) {
-		if (insulation > type.maxInsulation) {
-			return null;
+	public static ItemStack getIC2Cable(String type, int insulation) {
+		return getItem("cable", "type:"+type+",insulation:"+insulation);
+	}
+
+	public static ItemStack getItem(String name) {
+		return getItem(name, null);
+	}
+
+	public static ItemStack getItem(String name, String variant) {
+		ItemStack stack = IC2Items.getItem(name, variant);
+
+		if(stack==null || stack.isEmpty() || stack.getItem().getRegistryName().toString().equals("ic2:itemnouse")) {
+			// Treat IC2 Classic's ic2:itemnouse item as null.
+			stack = null;
```

Comment:
Any reason for null here, and not empty?

Reply:
There’s no reason why it needs to be null, all it needs to be is a placeholder of some sort, null is the most natural. But, empty would probably avoid NPEs in oredict registration.

Reply:
Yeah, im worried about null ItemStacks being passed into things they shouldn't. Returning an empty stack is better imo.

Reply:
Sounds good, in fact in my testing I got an NPE in oredict init from null itemstacks. However, it might throw on empty itemstacks as well...